management/commands/wait_for_db.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument("--poll_seconds", type=float, default=3)
+        parser.add_argument("--max_retries", type=int, default=60)
+
+    def handle(self, *args, **options):
+        max_retries = options["max_retries"]
+        poll_seconds = options["poll_seconds"]
+
+        for retry in range(max_retries):
+            try:
+                connection.ensure_connection()
+            except OperationalError as ex:
+                self.stdout.write(
+                    "Database unavailable on attempt {attempt}/{max_retries}:"
+                    " {error}".format(attempt=retry + 1,
+                                      max_retries=max_retries, error=ex)
+                )
+                time.sleep(poll_seconds)
+            else:
+                break
```

Comment:
It's good practice to log a success message when the database connection is successfully established. This can confirm that the command has completed successfully.